### ISSUE TITLE
check window_order unaffected in new dbplyr release

### ISSUE THREAD

**Issue by edward-burn** [darwin-eu-dev/omopgenerics#189]
Seeing this pr makes me wonder if we need to check if anything this line is going to be affected by the upcoming dbplyr release

https://github.com/darwin-eu-dev/omopgenerics/blob/26db2916ecb24c3101ae3908c6b98b2baf50de13/R/arrange.R#L23

**Comment by catalamarti** [comment]
yes, happy to explore this, I also think we have to document this properly
